README.rst

```diff
@@ -112,7 +112,7 @@ as well as all dependencies, can be done using pip:
 
 .. code-block:: bash
 
-    pip install quantum-jet
+    pip install git+https://github.com/XanaduAI/jet
```

Comment:
```suggestion
    pip install git+https://github.com/XanaduAI/jet.git
```
(is the `.git` needed?)

Reply:
Nope; however, the [PyPI docs](https://pip.pypa.io/en/stable/cli/pip_install/#vcs-support) do suggest adding an egg name as a query parameter.